srcpkgs/ffmpeg/template

```diff
@@ -99,7 +100,6 @@ do_configure() {
 		$(vopt_if webp '--enable-libwebp')
 }
 do_build() {
-	rm ${XBPS_WRAPPERDIR}/strip
```

Comment:
This was added in a0d88ba8496402b76ab503a59b3990e441f878ab for unknown reasons >.<

Reply:
W/o that line and `--disable-stripping`, the build will run into failure.